docs/SETTINGS.md

```diff
@@ -0,0 +1,195 @@
+# Repository Settings
+
+---
+
+These are the default settings that should be configured in each repo, changes can be made as needed.
+The settings described here should serve as a baseline, it is more important that there is some setting in place rather 
+than the exact settings being used across every repo.
+
+After a repo's settings are configured this file can be safely deleted from the repo since it is not needed for the
+project to run.
+
+## General
+
+---
+
+### Default Branch
+- Default branch should be `main`
+
+### Features
+- Enable `Issues` 
+- Enable `Projects` 
+
+`Wikis` and `Discussions` can be enabled if needed.
+
+### Pull Requests
+- Disable allow merge commits
+- Enable squash merging with default commit message set to `Pull request title`
+- Disable allow rebase merging
+- Enable always suggest updating pull request branches
+- Enable allow auto-merge
+- Enable automatically delete head branches
+
+## Collaborators and teams  
+
+---
+
+Every repo should have a team or set of teams configured in the `Manage Access` section. Ideally there should be
+two teams where one team is given `Admin` access and the other team is given either `Maintain` or `Write` access.
+
+The team responsible for code review should also be added to the [CODEOWNERS](../.github/CODEOWNERS) file so that
+they are automatically added as reviewers when a PR is opened. Read more on code owners [here](https://help.github.com/articles/about-codeowners/).
+
```

Comment:
Maybe add 

"Ask in the #help-github channel in Slack if you aren't clear on the correct teams to configure"

Reply:
Good call out, i'll add this